feature/gribi/ate_tests/weighted_balancing_test/port_flap_rebalanced_test.go

```diff
@@ -116,16 +117,21 @@ func TestPortFlap(t *testing.T) {
 		WithPersistence()
 	c.Start(ctx, t)
 	defer c.Stop(t)
+	defer func() {
```

Comment:
Since we already flush before test, why do we need to flush after test?

Reply:
just copying what is done for other tests. I also think flushing after test is not necessary, assuming all gribi tests do the same. will remove that and update the otg version as well.

Reply:
Should be fine even if the test does not clean up after itself. The binding is capable of doing a flush but it has to be requested in the binding.

https://github.com/openconfig/featureprofiles/blob/82732de3b989fa7c8399903e02ed2534f01ee6b3/topologies/binding/reset.go#L120

Reply:
great, was not aware of this.

Reply:
@liulk  addressed your comments, also applied the same missing change to TE-11.1

Reply:
@liulk could u please merge this pr?